### ISSUE TITLE
implement some measure methods

### ISSUE THREAD

**Issue by fengxiaohu** [xueleichen/PyTorch-Underwater-Image-Enhancement#5]
I don not see any measure methods such as PSNR in code.
So would you mind me adding PSNR to the code?
and how about plotting the loss figure, I want to add some code to retrieve loss value and plot it

**Comment by xueleichen** [comment]
Hi @fengxiaohu , I put the code for evaluation in another repo: https://github.com/xueleichen/PSNR-SSIM-UCIQE-UIQM-Python

As for loss plot, you can modify this line: https://github.com/xueleichen/PyTorch-Underwater-Image-Enhancement/blob/main/train.py#L55

You can store the loss in a local file or numpy array and then use matplotlib to plot the curve.

**Comment by fengxiaohu** [comment]
thank you,that really help me.
